observer-blindata-server/src/test/java/org/opendatamesh/platform/up/metaservice/blindata/ODMObserverBlindataAppIT.java

```diff
@@ -19,9 +19,7 @@
 import static org.assertj.core.api.Assertions.fail;
 
 @ExtendWith(SpringExtension.class)
-//@ActiveProfiles("test")
-//@ActiveProfiles("testpostgresql")
-//@ActiveProfiles("testmysql")
+@ActiveProfiles("test")
```

Comment:
Questo andrebbe lasciato commentato, perchè altrimenti se le pipeline di GitHub lanciano un comando per eseguire i test con uno specifico profilo viene ignorato e viene sempre considerato test come attivo. In questo specifico progetto va comunque bene perchè non esistono i profili testmysql e testpostgresql, quindi non è necessario, però tienilo a mente